spec/datasets/djornl/edge_type.yaml

```diff
@@ -23,3 +23,32 @@ oneOf:
   - const: AraNetv2-LC_lit-curated-ppi
     title: AraNetv2 literature-curated protein-protein interaction
     description: A layer of protein-protein interaction values from literature-curated small- to medium-scale experimental data; from the Arabidopsis AraNetv2 database. The LLS scores that serve as edge values were calculated to normalize the data for comparison across studies and different types of data layers (Lee et al, 2015).
+
+  - const: BIOGRID-ORGANISM-Arabidopsis-thaliana-Columbia-3-5-188-tab3_PPI
+    title: BIOGRID ORGANISM Arabidopsis thaliana Columbia 3.5.188 tab3 PPI
+    description: BioGRID interactions are relationships between two proteins or genes; the term “interaction” includes direct physical binding of two proteins, and co-existence in a stable complex and genetic interaction.  see https://wiki.thebiogrid.org/doku.php/experimental_systems
+
+  - const: AtPIN_PPI
+    title: AtPIN PPI
+    description: The interactions database includes all interactions present at the Arabidopsis thaliana Protein Interactome Database, the Predicted Interactome for Arabidopsis, Arabidopsis protein-protein interaction data curated from the literature by TAIR curators, BIOGRID and IntAct. https://atpin.bioinfoguy.net/cgi-bin/atpin.pl
+      
+  - const: mentha-AT-3702-040319_PPI
+    title: Mentha AT 3702 040319 PPI
+    description: Mentha archives evidence collected from different sources and presents these data in a complete and comprehensive way. Its data comes from manually curated protein-protein interaction databases that have adhered to the IMEx consortium and assigns to each interaction a reliability score that takes into account all the supporting evidence.  https://mentha.uniroma2.it/about.php
+
+  - const: aranetv2-subnet-AT-LC_PPI
+    title: AraNetv2 subnet AT-LC PPI
```

Comment:
Is this different from the existing network `AraNetv2-LC_lit-curated-ppi`?

Reply:
The two networks should be the same, or at least derived from the same source. The only difference might be if different filtering had been applied to either network.